src/ProjectSystemTools/ProjectSystemTools.csproj

```diff
@@ -34,6 +34,8 @@
     <PackageReference Include="Microsoft.VisualStudio.Utilities" Version="$(MicrosoftVisualStudioUtilitiesVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Validation" Version="$(MicrosoftVisualStudioValidationVersion)" />
     <PackageReference Include="System.Composition" Version="$(SystemCompositionVersion)" />
+    <PackageReference Include="System.Text.Json" Version="8.0.4"/>
+    <PackageReference Include="Microsoft.IO.Redist" Version="6.0.1"/>
```

Comment:
Pass in version via variable like other packages.

Reply:
done!